HeterogeneousCore/CUDAUtilities/interface/HistoContainer.h

```diff
@@ -73,7 +75,7 @@ namespace cudautils {
 } // namespace cudautils
 #endif
 
-template<typename T, uint32_t N, uint32_t M>
+template<typename T, uint32_t N, uint32_t M, uint32_t S=sizeof(T) * 8, typename I=uint32_t>
```

Comment:
can you document what the template parameters mean, and how one is supposed to use them ?